### DISCUSSION TITLE
Welcome to anomalib Discussions!

### DISCUSSION THREAD

**Discussion by samet-akcay** [open-edge-platform/anomalib#88] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alevangel** [comment]
Why I'm not able to start a new discussion? I cannot see the button "new discussion"

**Reply by djdameln** [reply]
Hmm, it seems that this feature is disabled at the organization level. Thanks for mentioning this! We weren't aware that users are not able to start a new discussion. We will try to solve this as soon as possible. In the mean time, feel free to ask your question in an issue. We could then convert it to a discussion manually as administrators.

**Reply by tominator95** [reply]
Are there any updates on this? It looks like others can start a Discussion? Unfortunately it does not work for me :-(

**Reply by samet-akcay** [reply]
@tominator95, no, this is still an issue we need to resolve. We thought that this had been enabled by the organisation, but the issue still persists. Now, only maintainers could create a discussion.

Meanwhile, you could keep creating issues that we could convert into discussion.

**Reply by samet-akcay** [reply]
@alevangel and @tominator95, just an update, creating a discussion has now been enabled!

**Reply by alevangel** [reply]
@samet-akcay thank you for the notification!

**Reply by hmf** [reply]
I am logged in GitHub but still have no "new discussion" button. On other projects I have access to this button. Appreciate any help. 
TIA

**Reply by ashwinvaidya17** [reply]
I'll try to contact the org admin to look into this issue
